### PR TITLE
Added Mobile menu for smaller screens

### PR DESCRIPTION
**Pull request by Beardless-sheik**
-Added index.js file with event listeners to toggle classes and display mobile menu
-Added styles in style.CSS to style the introduced mobile menu
-Added a Javascript linter for syntax check

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  document.querySelector('#mobile-menu').classList.toggle('show-mobile-menu');
+  //  document.querySelector('header').classList.toggle()
+}
+
+function removeMenu() {
+  document.querySelector('#mobile-container').classList.toggle('hide-mobile-menu');
+  document.querySelector('#mobile-menu').classList.toggle('show-mobile-menu');
+  //  ['mobile-backgrond-image-1','mobile-backgrond-image-2'].map( className =>
+  //  document.querySelector('#image2').classList.toggle(className));
+}
```
Comment (teekaytech) [inline index.js]: You have some commented lines of code in lines 4, 10, & 11. It is recommended that you always delete commented lines/blocks of code. If you need to keep track of such lines, you can do so with Git history.

Comment (teekaytech) [inline index.js]: - [ ] You have some commented lines of code in lines 4, 10, & 11. It is recommended that you always delete commented lines/blocks of code. If you need to keep track of such lines, you can do so with Git history.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @Beardless-sheik :wave:

_The main functionality of this feature seems to be working well. Your mobile menu looks good, your PR looks professional and all linter checks are running with no errors :clap: 
You're almost there :wink:_

__However, you need to make some required changes before moving on to the next assignment.__
> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it :blush: 

### **Required Changes** :recycle: 

- [ ] Kindly make the background of the mobile menu transparent, as observed in the Figma design.

| Figma Design | Your Design |
| ----------------- | ----------------- |
| original | clone |

- [ ] You might also want to reduce the vertical spaces between the menu items. 

- [ ] Check the comments under the review.

### **Optional Changes** :zap: 
_Every comment with the **OPTIONAL** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

**Comment by Beardless-sheik** [comment]
Thanks for the review. Changes done in the above 2 commits

**Review by Temmarie — APPROVED** [review]
## Status: Approved ✔️ 
### Hi @Beardless-sheik  , 
#### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  Congratulations! 🎉 

**Optional**
- [ ] Consider reducing the size of the cancel button in the dropdown menu to match the one in the Figma design.
#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**